modin/pandas/test/utils.py

```diff
@@ -120,6 +120,21 @@
     #     "col2": True,
     # },
 }
+# The parse_dates param can take several different types and combinations of
+# types. Use the following values to test date parsing on a CSV created for
+# that purpose at `time_parsing_csv_path`
+parse_dates_values_by_id = {
```

Comment:
nit: could also be named `parse_dates_id_to_value`.

Reply:
There was once a Python style guide tip at Google voting for names like this as opposed to `parse_dates_id_to_value` 😄 . One reason was that they compose better, e.g. `parse_dates_values_by_id_by_name['id']['name']` is easier to read than `id_to_name_to_parse_dates['id']['name']` because in the former, the index names in the variable name appear in the same order as the index names in the brackets.